server/src/services/discord.service.ts

```diff
@@ -45,21 +45,33 @@ class DiscordService {
 
   }
 
+  public async getChannelInfo(channelId: string) {
+
+    return await this.client.channels.fetch(channelId);
+
+  }
+
   public async sendDiscordMessage(channelId: string, payload: any) {
 
     try {
+      logger.info('GENERATING EMBED');
       const dataToSend = new MessageEmbed()
         .setColor('#0099ff')
         .setAuthor({
           name: payload.network?.name,
           iconURL: (payload.network?.favicon as Types.Image)?.urls?.small,
-          url: `https://${payload.network.domain}`,
+          url: `https://${payload.network?.domain}`,
         })
-        .setThumbnail(payload.member.profilePicture.url)
+        .setThumbnail(payload.member?.profilePicture?.url || null)
         .setTimestamp()
-        .setFooter({ text: `Space: ${payload.space.name}`, iconURL: payload.space.banner?.url });
+        .setFooter({
```

Comment:
Let's follow the same patter an Slack. If the space is not there, we shouldn't show anything.